resources/scripts/openapi-icon-tool/src/main/java/IconTool.java

```diff
@@ -0,0 +1,36 @@
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+
+public class IconTool {
+    static String iconsSourcePath;
+    static String openApiConnectorsPath;
+    public static void main(String[] args) throws IOException {
+        iconsSourcePath = Utils.getConfigValue("iconsSourcePath");
+        openApiConnectorsPath = Utils.getConfigValue("openApiConnectorsPath");
+        List<String> ballerinaPackages = Utils.findBallerinaPackages(openApiConnectorsPath);
```

Comment:
We need to update the icon of all connectors to what we have in `iconsSourcePath ` with this tool I guess. Does this util function return all the connector packages we have or only changed ones? 

Or does this tool is updating icons of changed connectors only?

Reply:
It updates all the icons

Reply:
+1. Then that would be fine.